Outlands_Webminister_Handbook.md

```diff
@@ -35,30 +41,31 @@ The kingdom website shall maintain a library of current governing documents and
 ### 3.1 Office Hierarchy
 #### 3.1.1 Structure of the Office of Web Ministry
 The Office of Web Ministry is a Greater Office of the Kingdom of the Outlands. It comprises the kingdom webminister, associated kingdom and special project deputies, and branch webministers. 
+TODO why is it a Greater Office, should it be, is this in scope?
 
-##### Kingdom Webminister (Webwryght)
+##### Kingdom Webminister
 The Kingdom Webminister serves as the liason between kingdom officers, the populace, and the Office of Web Ministry. They are responsible for overseeing management of all official web properties of the Outlands and local groups. 
 
 The Kingdom Webminister reports to the Society Webminister.
 
-##### Northern Groups Deputy (Waynwryght)
-Addresses concerns from branches X,Y,Z
-Ideally also a branch webminister
+##### Northern Groups Deputy
+Addresses concerns from branches X,Y,Z - TODO list
+Ideally also a branch webminister in one of those branches
 
-##### Southern Groups Deputy (Plowwryght)
-Addresses concerns from branch X,Y,Z
-Ideally also a branch webminister
+##### Southern Groups Deputy
+Addresses concerns from branch X,Y,Z - TODO list
+Ideally also a branch webminister in one of those branches
 
-##### Deputy for Outlands.org (Botwryght)
+##### Deputy for Outlands.org
 Manages content requests for Outlands.org
 
 The Kingdom Webminister is ultimately responsible for all web content which pertains to the Kingdom of the Outlands.
 
-##### Deputy for the Calendar (Whelwryght)
+##### Deputy for the Calendar
```

Comment:
This is a discussion in flight with the seneschal - Calendar currently falls there.